.github/workflows/upload-wasm08-clients.yaml

```
@@ -0,0 +1,88 @@
+name: Upload wasm08 clients as Github workflow artifact
+on:
+  pull_request:
+    paths:
+      - .github/workflows/upload-wasm08-clients.yaml
+      - Cargo.toml
+      - ci/**
+      - ibc/**
+      - ibc-core/**
+      - ibc-apps/**
+      - ibc-data-types/**
+      - ibc-clients/**
+      - ibc-primitives/**
+      - ibc-query/**
+      - ibc-testkit/**
+      - ibc-derive/**
+  push:
+    branches: main
+    paths:
+      - .github/workflows/upload-wasm08-clients.yaml
+      - Cargo.toml
+      - ci/**
+      - ibc/**
+      - ibc-core/**
+      - ibc-apps/**
+      - ibc-data-types/**
+      - ibc-clients/**
+      - ibc-primitives/**
+      - ibc-query/**
+      - ibc-testkit/**
+      - ibc-derive/**
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  upload-tm-wasm08-client:
```

```suggestion
  upload-tendermint-cw-client:
```

This should be adjusted for other places where `wasm08` used. Actually, the `08-wasm` is a distinct type of light client that enables operating the CosmWasm contract of other clients like in our case the CosmWasm contract of the Tendermint.